### PR TITLE
Feature/vpn manager setup

### LINKED ISSUE #678: @/Outline VPN Set up
- Create an Outline VPN management app to manage, sending, resending of user VPNs.
- Implement this using Next js and Material UI
- The set up needs to put in mind handing of statistics

### PR DESCRIPTION
**Pull request by koechkevin**
## Description

This PR sets up vpn manager using material UI and Next js.
- Adds an execution script to read users from spreadsheet given and converts the data to JSON. `pnpm process-new-hires`.
- Save the credentials provided [here](https://vault.codeforafrica.org/#/vault?search=VPN&organizationId=unassigned&itemId=474a48ad-2c80-4383-85a7-33ef627e771a) in a credentials.json.
- Updatr `.env` as per `.env.example`

Fixes #678 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: apps/vpnmanager/package.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    "@mui/material": "^5.14.20",
+    "@mui/utils": "^5.14.20",
+    "@sentry/nextjs": "^7.105.0",
+    "@svgr/webpack": "^8.1.0",
+    "@types/jest": "^29.5.12",
+    "googleapis": "^133.0.0",
+    "jest": "^29.7.0",
+    "next": "14.1.3",
+    "react": "^18",
+    "react-dom": "^18",
```
Comment (kilemensi) [inline apps/vpnmanager/package.json:30]: Use the same exact version pinning as in charterafrica, codeforafrica, etc. We don't want to end up with different versions.

File: apps/vpnmanager/.env.template [hunk 2]
```diff
@@ -0,0 +1,6 @@
+NEXT_APP_GOOGLE_CREDENTIALS=/path/to/credentials.json
+NEXT_APP_GOOGLE_SHEET_ID=
+NEXT_APP_GOOGLE_SHEET_RANGE=New Hires!A:Z
+NEXT_APP_VPN_API_URL=
+SENTRY_AUTH_TOKEN=
+NEXT_APP_SENTRY_DSN=
```
Comment (kilemensi) [inline apps/vpnmanager/.env.template:6]: Is the `NEXT_APP` prefix what we used in other apps as well?

File: apps/vpnmanager/next.config.mjs [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import { withSentryConfig } from "@sentry/nextjs";
```
Comment (kilemensi) [inline apps/vpnmanager/next.config.mjs:1]: Does this **need** to be `.mjs`?

File: package.json [hunk 4]
```diff
@@ -10,6 +10,7 @@
     "build": "turbo run build",
     "build-next": "turbo run build-next",
     "build-payload": "turbo run build-payload",
+    "build-ts": "turbo run build-ts",
```
Comment (kilemensi) [inline package.json:13]: Shouldn't this be `build` i.e. in vpnmanager, build will call build-ts but in other apps, it'll just call build.

File: docker-compose.yml [hunk 5]
```diff
@@ -50,6 +50,19 @@ services:
...
+  vpnmanager:
+    build:
+      context: .
+      dockerfile: Dockerfile.vpnmanager
+      args:
+        SENTRY_ORG: ${SENTRY_ORG}
+        SENTRY_ENV: ${SENTRY_ENV}
+        SENTRY_PROJECT: ${SENTRY_PROJECT}
+        NEXT_APP_SENTRY_DSN: ${NEXT_APP_SENTRY_DSN}
+        SENTRY_AUTH_TOKEN: ${SENTRY_AUTH_TOKEN}
```
Comment (kilemensi) [inline docker-compose.yml:62]: Set sentry env to local... this file should only be used for local development.

File: apps/vpnmanager/src/layout.tsx [hunk 6]
```diff
@@ -0,0 +1,25 @@
+import type { Metadata } from "next";
```
Comment (kilemensi) [inline apps/vpnmanager/src/layout.tsx:1]: Is this file being used anywhere?

**Review by kilemensi — COMMENTED** [review]
👍🏽 


----

Next.js now has automatic font [loading/optimisation](https://nextjs.org/docs/app/building-your-application/optimizing/fonts). We shouldn't need to manually download anymore. (Unless you've noticed some issues).

File: apps/vpnmanager/next.config.mjs (hunk 3 above)
Comment (koechkevin) [reply apps/vpnmanager/next.config.mjs:1]: @kilemensi it doesn't have to be, I thought  using `import/export` is cooler than `require/module.exports`

**Comment by koechkevin** [comment]
@kelvinkipruto Have a look at this

**Review by kelvinkipruto — APPROVED** [review]
LGTM

**Pull request merged**